### ISSUE TITLE
Add some example CSS to illustrate some best practices on accessible styling

### ISSUE THREAD

**Issue by mefechoel** [mefechoel/svelte-navigator#11]
Some CSS should be added to the examples, to show some basic techniques on how to style an app in an accessible way.

Focus outlines should be explained, as they are very relevant to the project (see #10).

Maybe Custom inputs, e.g. checkboxes, since they are very easy to make inaccessible.
Maybe add some more information and linked articles to the readme.
